hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -745,7 +745,7 @@ objects:
               index: rh_osd_malware_scan
               whiteList: \.log$
               sourceType: openshift:debug
-            - path: /host/var/lib/kubelet/pods/*/volumes/kubernetes.io~*/pvc-*/backplane-audit.log
+            - path: /host/var/lib/kubelet/pods/*/volumes/kubernetes.io~*/pvc-*/mount/backplane-audit.log
```

Comment:
Is this needed? For commercial, the SSS targeting commercial doesn't contain the mount path, and I still see recent events from management clusters in our Splunk query. 

https://github.com/openshift/splunk-forwarder-operator/blob/master/hack/olm-registry/olm-artifacts-template.yaml#L596C2-L599C32 

https://osdsecuritylogs.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22openshift_managed_backplane_prod%22%20infraID%3D%22hs-mc*%22&display.page.search.mode=smart&dispatch.sample_ratio=1&workload_pool=standard_perf&earliest=-30m%40m&latest=now&sid=1749232932.27223